.semaphore/semaphore.yml

```diff
@@ -19,3 +19,5 @@ blocks:
           commands:
             - checkout
             - make verify
+	    - go get ./...
```

Comment:
Is this a tab?

Reply:
Oops, my IDE played a trick on me; fixing.